src/test/project.test.js

```diff
@@ -0,0 +1,24 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import fs from 'fs';
+import _ from 'lodash';
+
+describe( 'RN version from root is same in', () => {
+	const rootRNVersion = parseRNVersionFromPackageJson( '.', 'dependencies' );
+	expect( rootRNVersion ).toBeDefined();
+
+	[ 'react-native-aztec', 'react-native-gutenberg-bridge' ].forEach( ( path ) => {
+		it( path, () => {
+			const otherRNVersion = parseRNVersionFromPackageJson( path, 'peerDependencies' );
+			expect( otherRNVersion ).toEqual( rootRNVersion );
+		} );
+	} );
+} );
```

Comment:
:wave: @hypest! I'm curious if you have any thoughts about how I'm using a jest test to enforce that the RN versions are consistent across the different package.json files. This feels rather hacky to me, but I assume we would always want these to be consistent, and I couldn't think of a better way to handle it. WDYT?